tests/subscriber.coffee

```diff
@@ -179,19 +179,19 @@ describe 'Subscriber', ->
         it 'should not remove subscription on an unexisting subscription', (done) =>
             subscriber = new Subscriber(@subscriber.redis, 'invalidid')
             subscriber.removeSubscription @testEvent, (removed) =>
-                should.not.exist removed
+                should.exist(removed)
```

Comment:
Ok got it, so the `removed` variable should be renamed `err` or `errorDeleting`.